scripts/check_rule_titles.js

```diff
@@ -0,0 +1,106 @@
+const fs = require('fs');
+const path = require('path');
+const fetch = require('cross-fetch');
+const { Command } = require('commander');
+const YAML = require('yaml');
+const { parse } = require("csv-parse/sync");
+
+const program = new Command();
+
+const RULES_DIR_PATH = "./rules"
+
+async function main() {
+    program.name('check-rule-titles')
+    .description('Check rule title conformity with existing rule titles')
+    .version('1.0.0', '-v, --version')
+    .option(
+      '--csv-delimiter',
+      'csv delimiter to use',
+      ',',
+    )
+    .action(async function(){
+      data = await load()
+      rulesHash = await buildRulesHash(data)
+      results = await checkTitles(rulesHash)
+      console.log(results)
+    });
+
+    await program.parseAsync(process.argv);
+}
+
+async function load() {
+  const response = await fetch("https://docs.bearer.com/rules.csv")
```

Comment:
We should really check for local stuff this is kinda deceptive since its just whatever has been released so far and puts a dependancy on the docs site.

Reply:
^ Will remove for now and re-think solution